src/parameters/all_parameters.cpp

```diff
@@ -273,10 +273,14 @@ void AllParameters::declare_parameters (dealii::ParameterHandler &prm)
                       dealii::Patterns::Bool(),
                       "Outputs the high-order mesh vtu files. False by default");
 
-    prm.declare_entry("enable_higher_order_vtk_output", "false",
+    prm.declare_entry("enable_higher_order_vtk_output", "true",
                       dealii::Patterns::Bool(),
-                      "Enable writing of higher-order vtk files. False by default. If modified to true,"
-                      "number of subdivisions will be chosen according to the max of grid_degree and poly_degree.");
+                      "Enable writing of higher-order vtk files. True by default; "
+                      "number of subdivisions is chosen according to the max of grid_degree and poly_degree.");
+
+    prm.declare_entry("output_face_results_vtk", "false",
```

Comment:
I like this change, as the `surface_solution` files often aren't very useful. However, could you set the parameter to `true` on the airfoil tests? It's nice to be able to plot surface pressure without additional postproccesing.

Reply:
fixed